res/css/views/settings/_LayoutSwitcher.pcss

```diff
@@ -28,6 +28,7 @@ limitations under the License.
         flex-shrink: 1;
         display: flex;
         flex-direction: column;
+        overflow: hidden;
```

Comment:
Why this change?

Reply:
There was an issue with the background bleeding over the border radius. And `overflow: hidden` fixed the issue in this context. Weirdly happened in only one browser

Reply:
Is this specific to the colour pass?

Reply:
Yes, i have not seen this issue occur in mainline. But i have not looked for the root cause.